### PR TITLE
minimal changes to get things to compile.

### PR DESCRIPTION
**Pull request by Carreau**
I couldn't get this to compile without that. 

Am I missing some obvious step (running for a plane, so will be offline for monday)

### CODE REVIEW

**Comment by jasongrout** [comment]
Thanks for looking into this!

@blink1073 - does your PR compile?  I notice that some of these files are the same as some of your files (like kernel.ts), and we want the best from both PRs.

@Carreau - can you resubmit this to the jupyter/phosphor-notebook repo?

**Comment by blink1073** [comment]
I am in the process of gutting my PR to use Phosphor Signals and the Futures concept we discussed on Gitter.

File: gulpfile.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ gulp.task('clean', function(cb) {
 gulp.task('src', function() {
     var project = typescript.createProject({
         declarationFiles: true,
-        noImplicitAny: true,
+        noImplicitAny: false,
```
Comment (blink1073) [inline gulpfile.js]: In my opinion we want to keep this on.  What do you say @ccolbert?

**Comment by blink1073** [comment]
@jasongrout, I think if you merge this I'd have to just squash all my commits into one or get stuck in conflict hell on a rebase.

File: gulpfile.js (hunk 1 above)
Comment (Carreau) [inline gulpfile.js]: Agreed in the long run to get no implicit any, but in the short term to get things to compile...

**Comment by jasongrout** [comment]
@blink1073 - I would rather glean from this PR any useful fixes to help you in your PR than create massive conflicts for you.

**Comment by blink1073** [comment]
I'll review it with that in mind.

File: src/base/js/utils.ts [hunk 2]
```diff
@@ -0,0 +1,860 @@
...
+// Distributed under the terms of the Modified BSD License.
+
+
+/// <amd-dependency path="codemirror/mode/meta" name="CodeMirrorModeMeta"/>
+/// <amd-dependency path="codemirror/lib/codemirror" name="CodeMirror"/>
+import $ = require('jquery')
+import moment = require('moment')
+
+
+export interface MathJax {
```
Comment (blink1073) [inline src/base/js/utils.ts]: in my PR I removed any util functions that were not explicitly used in the current set of files.

Comment (Carreau) [inline src/base/js/utils.ts]: These are just the IPython version of the js files typescriptified, as I used them in My ts branch as an alternate for the  IPython JS files. Feel free to pick removed any pieces you like/don't like. I just did af fast as could while beeing still online to get the things to compile. I didn't really paid attention to what was hapenning.

Comment (blink1073) [inline src/base/js/utils.ts]: Ah, I believe I started from these very files in my PR :smile:, so thanks!

Comment (blink1073) [inline src/base/js/utils.ts]: As for kernel.ts, I believe I've made the same changes you have, plus more, so I would recommend we close this PR in favor of #2.

**Pull request closed without merging**